### PR TITLE
quickbuild bug fix

### PR DESCRIPTION
**Pull request by S-Aoi**
Now `@mxmlc` can receive `+configname=air` argument. And paths can be quoted with `"`.

### CODE REVIEW

File: External/Plugins/AS3Context/Compiler/FlexShells.cs [hunk 1]
```diff
@@ -321,6 +321,10 @@ public void QuickBuild(FileModel theFile, string flex2Path, bool requireTag, boo
                                             }
                                         }
                                         isPath = true;
+                                        // remove quotes
+                                        if (arg.StartsWith("\"") && arg.StartsWith("\""))
```
Comment (Gama11) [inline External/Plugins/AS3Context/Compiler/FlexShells.cs]: Isn't the second condition here supposed to use `EndsWith()`?

**Comment by elsassph** [comment]
Why removing quotes exactly?

**Comment by S-Aoi** [comment]
Some paths may contain spaces. When I quote the path with `"`, FD show me an error saying `This file is not a valid AS3 class or does not include a valid @mxmlc build command.`.

**Comment by elsassph** [comment]
Oh sorry, you're right there was a problem when your mxmlc command has quotes already.

**Pull request merged**